otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css

```diff
@@ -313,10 +313,10 @@ body {
 }
 .docs-sidebar-toc a {
   text-decoration: none;
-  color: var(--main-body-color);
+  color: var(--dt-color-black);
 }
 .docs-sidebar-toc a:hover {
-  color: var(--dt-color-blue-50);
+  color: var(--dt-color-primary-hover);
```

Comment:
please not this color - it is far away from being attractive

Reply:
blue should be fine, but not magenta. I do not really like it at all

Reply:
that is not an enforcement to do it this way. I.e. Yam is not using it this way (on sidebars). It doesn't look nice. Especially on the right sidebar this is really ugly.